src/main/java/core/basesyntax/service/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Inventory;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements FruitOperationHandler {
```

Comment:
The class name `BalanceOperation` should clearly reflect its purpose. Consider renaming it to match the operation it handles, like `BalanceHandler` to adhere to the naming conventions.